src/server/api/controllers/questions.controller.js

```diff
@@ -0,0 +1,14 @@
+const knex = require('../../config/db');
+
+const createQuestion = async (body) => {
+  await knex('questions').insert({
+    question: body.question,
+  });
```

Comment:
This could also be written like this, just fyi :) 
```suggestion
const createQuestion = async ({ question }) => {
  await knex('questions').insert({ question });
```